server/controllers/user.js

```diff
@@ -21,3 +24,42 @@ exports.searchUsers = asyncHandler(async (req, res, next) => {
 
   res.status(200).json({ users: users });
 });
+
+exports.uploadImage = asyncHandler(async (req, res, next) => {
+  if (!req.file) {
+    throw new Error("No file provided!");
+  }
```

Comment:
If I'm correct, this will throw a 500 status code? That's not the proper response code. We should set the status to 400 Bad Request